tests/lib/rook/upstream.py

```diff
@@ -83,13 +85,15 @@ def build(self):
             )
 
             logger.info("[build_rook] Tag image")
-            execute('docker tag "rook-build/ceph-amd64" rook/ceph:master')
+            execute('docker tag "rook-build/ceph-amd64" rook/ceph:%s'
+                    % settings.UPSTREAM_ROOK.VERSION)
 
             logger.info("[build_rook] Save image tar")
             # TODO(jhesketh): build arch may differ
             execute(
-                "docker save rook/ceph:master | gzip > %s"
-                % os.path.join(self.builddir, 'rook-ceph.tar.gz')
+                "docker save rook/ceph:%s | gzip > %s"
+                % (settings.UPSTREAM_ROOK.VERSION,
```

Comment:
Lets tag this with something like `rookcheck`, and then modify the yaml to point to `rook/ceph:rookcheck`. That way we'll guarantee that we're always running with the version that we've built.

Reply:
I used the version and rookcheck suffix like like `rook/ceph:v1.4.5-rookcheck`